test/sse_proxy_handler_test.dart

```diff
@@ -128,8 +128,7 @@ void main() {
     await webdriver.get('http://localhost:${proxy.port}');
     await serverSse.connections.next;
     expect(serverSse.numberOfClients, 1);
-
-    await webdriver.get('chrome://version/');
+    await webdriver.quit();
```

Comment:
This "works" but not sure if this preserves the intention of the test anymore? 
"Disconnects when navigating away" is the name of the test, but navigating to any other page, adding a wait after navigating, pumping the event queue all result in there still being 1 connection. I assume this is a change in the newer sse package when running under Dart 3?

Reply:
Yeah I think this is probably fine 👍